File: port/csharp/jane/handler/testserver/TestEmptyHandler.cs
using Jane.Bean;

namespace Jane.handler.testserver
{
	public class TestEmptyHandler
	{
		/*\
		\*/

		public static void onProcess(NetManager manager, IBean _arg_)
		{
			TestEmpty arg = (TestEmpty)_arg_;
			// System.Console.WriteLine("{0}.onProcess: arg={1}", GetType().Name, arg);

		}
	}
}

File: src/jane/handler/testserver/TestEmptyHandler.java
package jane.handler.testserver;

import jane.bean.TestEmpty;
import jane.core.BeanHandler;
import jane.core.Log;
import jane.core.NetManager;
import org.apache.mina.core.session.IoSession;

public final class TestEmptyHandler implements BeanHandler<TestEmpty>
{
	@Override
	public TestEmpty beanStub()
	{
		return TestEmpty.BEAN_STUB;
	}

	/*\
	\*/

	@Override
	public void onProcess(final NetManager manager, final IoSession session, final TestEmpty arg)
	{
		Log.debug("{}.onProcess: arg={}", getClass().getName(), arg);
	}
}
